### ISSUE TITLE
[Feature] Option to lockdown romm and disable option to delete/upload

### ISSUE THREAD

**Issue by NaokiNXN** [zurdi15/romm#108]
Hi I have this setup and its great for personal use and would like to share it with some friends but worried they may accidentally delete or press an option I don't want them to press.

If an environment variable could be added to disable the option to delete/upload files this would be great, I can then change this if needed at any point by changing the variable.

**Comment by NaokiNXN** [comment]
Apologies had seen that post before but didn't see the conversation about blocking the ability to delete.
Regarding this point user management would need a lot of work as mentioned in that issue thread but thought id raise this one as seems adding a variable to disable the delete & upload buttons would likely be a lot easier and could act as a quick stop gap for those looking for a bit of extra protection when sharing it.

**Comment by crispybegs** [comment]
> Apologies had seen that post before but didn't see the conversation about blocking the ability to delete. Regarding this point user management would need a lot of work as mentioned in that issue thread but thought id raise this one as seems adding a variable to disable the delete & upload buttons would likely be a lot easier and could act as a quick stop gap for those looking for a bit of extra protection when sharing it.

no worries. it's def a user-level permissions issue, otherwise how can you know when other people are accessing your library at the time you want to enable your buttons to do some admin. you'd be hopping in and out of your env every time you want to make any small change. Doesn't seem practical really.

**Comment by NaokiNXN** [comment]
True, although in fairness if the buttons could be disabled I would probably just manually delete the files if needed and then do a new scan. But as you said not an ideal solution.

**Comment by zurdi15** [comment]
> we've had this here - #50 but yes, agree that it's a much-needed option to open it up to the internet, even for personal use. it's local network only at the moment, even if you're flying solo

As @crispybegs pointed, I am closing this issue due to the priority of developing a proper user-level permissions system. I think is not worth it to put effort in a workaround like this one.
